src/profile-logic/marker-timing.js

```diff
@@ -104,7 +104,8 @@ export function getMarkerTiming(
         if (otherStart > marker.start + marker.dur) {
           break;
         }
-        if (otherEnd > marker.start) {
+        // We use + 1 to account for possible radius
```

Comment:
I believe this will break the layout of flame charts for UserTimings. Also I think caused the layout to not work correctly. See my comment at the end.